src/pr-info.ts

```diff
@@ -388,7 +388,6 @@ const configSuspicious = <ConfigSuspicious>(async (path, getContents) => {
         suspect = tester(text);
     } else {
         const oldText = await getContents("master");
-        if (!oldText) return "created";
```

Comment:
Removing this changes other places — at least the test for `tsconfig.json` should return `"created"` if there is no old version.  (I don't remember if that's the only place.)

Reply:
Done :heavy_check_mark: